docs/specs/signatures/description.markdown

```diff
@@ -0,0 +1,383 @@
+      BIP: ?
```

Comment:
Most of the markdown files in the repo have `.md` filename suffixes, rather than `.markdown` (except for the Web site).